changelog.d/169.trivial.rst

```diff
@@ -0,0 +1,17 @@
+Create semver package and split code among different modules in the packages.
```

Comment:
I think, these changes do not do justice to the word "trivial". :laughing: Could we rename the filename to `changelog.d/169.feature.rst`? :wink:

Reply:
But it doesn't really add anything visible to the end user. Although I guess the change of location of the cli functions does make it different. This doesn't really feel like a feature. Maybe we should add an `internal` category?